woocommerce/woocommerce
# Add product variation navigation component

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/35991






### How to test the changes in this Pull Request:



1. Create a product with some variations using the legacy experience and grab its id and the first variation id
2. Got to `/wp-admin/admin.php?page=wc-admin&path=/product/{productId}/variation/{variationId}`
3. A navigation bar should be shown at the bottom of the page
4. If visiting the first variation then the back button should not allow navigation and looks like disabled
5. After clicking next the back button should be active and should let you navigate back to the prev variation. Also you should be shown the following variation.
6. When landing to the last variation the next button should not allow navigation and looks like disabled
7. If click the action/middle button you should navigate to the parent product

<video src="https://user-images.githubusercontent.com/13334210/208463460-b88fcc8c-fba0-4a52-920e-1b12ef0479f4.mov"></video>



### Other information:




### FOR PR REVIEWER ONLY:
